src/lib/secretGen.js

```diff
@@ -16,7 +16,11 @@ module.exports = function(options = { app: '', region: '', env: '', name: '' })
     if(typeof options === 'string')
         return [env.getSecretPrefix() || '', options].join('/');
 
-    const prefix = [options.app, options.region, options.env].map(a => a.toLowerCase()).join('/');
+    let prefix = [options.app, options.region, options.env].map(a => a.toLowerCase()).join('/');
```

Comment:
You could also just do `[options.app, options.region, options.env, options.name].filter(a => a).map(a => a.toLowerCase()).join('/');` couldn't you?

Reply:
We could, but there was no comment on why he was only adding the name later on if `prefix.length > 3` so I felt this was less likely to change the overall behavior.  He is only adding in the name after he verifies the length is 3 where as using the filter with them all up front may require an adjustment to that check (not sure since I do not know the significance of length > 3).

I can make the change if we want to, just wasn't sure if it was going to cause an issue else where since I am not familiar with the why.